robot-core/src/main/java/org/obolibrary/robot/DiffOperation.java

```diff
@@ -151,6 +174,7 @@ public static boolean compare(
    * @param axiom a string representation of an OWLAxiom
    * @return a string with labels inserted next to IRIs
    */
+  @Deprecated
```

Comment:
The `@Deprecated` methods should also have a `@deprecated` Javadoc tag

Reply:
Okay.

Reply:
Addressed in 26ede14fbd1e9f655672325f61541a668a6049d4.